app/src/main/java/com/rafraph/pnineyHalachaHashalem/myAudio.java

```diff
@@ -360,13 +621,31 @@ public void skip_to_previous(View view) {
         buttonPlayPause.setImageResource(R.drawable.baseline_pause_circle_outline_white_48);
         Intent broadcastIntent = new Intent(Broadcast_SKIP_PREVIOUS);
         sendBroadcast(broadcastIntent);
+        broadcastIntent = new Intent(Broadcast_Speed);
+        int choice=spinner.getSelectedItemPosition();
+        if(choice==0)
+            speed=2f;
+        if(choice==1)
+            speed=1.5f;
+        if(choice==2)
+            speed=1f;
+        if(choice==3)
+            speed=0.75f;
+        broadcastIntent.putExtra("speed",speed);
+        broadcastIntent.putExtra("play",0);
+        sendBroadcast(broadcastIntent);
+        txtI.setText(book_name + " " +   convert_character_to_id(chapter)+", "+convert_character_to_id(section-1));
+        if(section+1486<'א')

```

Comment:
Where the section variable is defined?
What is the 1486?

Reply:
הוספתי הסבר. זה פשוט נועד לתקן מקרה קצה שרשום שם תת במקום א